tests/runtimes/simple-keyvalue/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "simple-keyvalue-runtime"
+version = "0.1.0"
+authors = ["Oasis Protocol Foundation <info@oasisprotocol.org>"]
+edition = "2018"
+license = "Apache-2.0"
+
+[dependencies]
+oasis-runtime-sdk = { git = "https://github.com/oasisprotocol/oasis-sdk", branch = "main" }
```

Comment:
This should reference the crate by relative path instead of Git. Otherwise this will not use the correct version.